addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -139,6 +139,11 @@ $-editor-messages-margin-x: 2%;
 [draggable] {
     user-select: none;
 }
+/* Prevent the native drag and drop for images. */
+img {
+    -moz-user-drag: none;
+    -webkit-user-drag: none;
+}
```

Comment:
The other alternative would be to use [the draggable attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable) but it would require a lot of changes... (add attribute on edition start or upgrade script, change every image template or have an observer that add the attribute on every images...)